CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.0-nullsafety.1
```

Comment:
Sorry one more - lets go ahead and make this the official `1.0.0` release. Please update the sdk constraint minimum to `>=2.12.0 <3.0.0` as well.

I could follow up with this change if you prefer as well.